qa/build.xml

```diff
@@ -21,55 +21,49 @@
 -->
 <project name="JBossTS QA Suite Distribution" default="dist">
   <property environment="env"/>
-  <property name="driver.home" value="dbdrivers"/>
   <property name="orbtype" value="idlj"/>
   <!-- set this property to the location of a JBossTS JTA/JTS build or installation -->
   <property name="org.jboss.jbossts.qa.ts.home" location="dist"/>
   <property name="narayana.version" value="5.11.3.Final-SNAPSHOT"/>
-  <!-- you should not need to configure anything below here -->
+
   <!--
-        This is a redhat vpn server. for non-vpn use, mirror it locally or copy the drivers by hand.
-        wget \-\-no-parent -m http://www.qa.jboss.com/jdbc-drivers/
-        wget \-\-no-parent -m http://www.qa.jboss.com/jdbc-drivers/jConnect-6_0/classes
-        ant -Ddriver.url=file:///path/to/www.qa.jboss.com/jdbc-drivers get.drivers
+        The previous logic used to download drivers has been replaced for
```

Comment:
Just a personal opinion. To me it's not necessary to have this "long" explanation that there was some old functionality. All information about the any prior handling is easily available from version control system.
It could be fine to just say the drivers are downloaded from maven repos.
From what I'm a little bit missing is the information to what the drivers are used to. What does mean "QA tests against ...". Here I think it could be beneficial to have information that databases are for running the object stores.